walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,96 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    Collection<TimeRange> BusyTimes= new ArrayList <TimeRange>();
```

Comment:
nit: I see some linting errors; you could add the following auto-formatting tool to fix them for you. 
Just add the following plugin into the pom.xml for this project (~step/walkthroughs/week-5-tdd/project/) under the <plugins> tag:
![image](https://user-images.githubusercontent.com/55038687/85306680-129d7c80-b46c-11ea-92e2-7adc18c6605b.png)

After that you can just run the command 'mvn fmt:format' inside the same directory the pom.xml file is in.

Reply:
(here's a friendly copy-paste):

 <plugin>
        <groupId>com.coveo</groupId>
        <artifactId>fmt-maven-plugin</artifactId>
        <version>2.9</version>
        <configuration>
          <verbose>true</verbose>
        </configuration>
        <executions>
          <execution>
            <goals>
              <goal>check</goal>
            </goals>
          </execution>
        </executions>
      </plugin>

Reply:
Will do!